README-SW.md

```diff
@@ -1,7 +1,7 @@
-<!-- Do not translate this -->
+<!-- Tafadhali usitafsiri hii -->
 <details>
 <summary>
-<strong> Soma mwongozo huu kwa lugha nyingine </strong>
+<strong> Tafsiri mwongozo huu kwa lugha nyingine </strong>
```

Comment:
the translation `Tafsiri mwongozo huu kwa lugha nyingine` should be changed to => `Soma mwongozo huu katika lugha zinginezo` 

like in the English version *Read this guide in other languages* when `tafsiri` is used it would bring a diferent meaning from that.